samples/http-cloudevents/client.py

```diff
@@ -21,44 +21,34 @@
 
 
 def send_binary_cloud_event(url):
-    # define cloudevents data
+    # This data defines a binary cloudevent
     attributes = {
-        "id": "binary-event-id",
-        "source": "localhost",
-        "type": "template.http.binary",
-        "dataontenttype": "application/json",
-        # Time will be filled in automatically if not set
-        "time": "2018-10-23T12:28:23.3464579Z",
+        "type": "com.example.sampletype1",
+        "source": "/mycontext",
     }
-    data = {"payload-content": "Hello World!"}
+    data = {"message": "Hello World!"}
 
-    # create a CloudEvent
     event = CloudEvent(attributes, data)
     headers, body = event.to_http(converters.TypeBinary)
 
     # send and print event
-    requests.post(url, headers=headers, json=body)
+    requests.post(url, data=body, headers=headers)
     print(f"Sent {event['id']} from {event['source']} with " f"{event.data}")
 
 
 def send_structured_cloud_event(url):
-    # define cloudevents data
+    # This data defines a binary cloudevent
     attributes = {
-        "id": "structured-event-id",
-        "source": "localhost",
-        "type": "template.http.structured",
-        "datacontenttype": "application/json",
-        # Time will be filled in automatically if not set
-        "time": "2018-10-23T12:28:23.3464579Z",
```

Comment:
A good follow-up might be a sample showcasing all the optional properties.

Reply:
I'll probably make that a follow up PR for another sample code sample.